src/main/kotlin/no/nb/mlt/wls/infrastructure/synq/SynqOrderPayload.kt

```diff
@@ -21,8 +21,7 @@ data class SynqOrderPayload(
     val priority: Int,
     val owner: SynqOwner,
     val orderLine: List<OrderLine>,
-    val customer: String? = null,
-    val shippingAddress: ShippingAddress? = null
+    val customer: String? = null
```

Comment:
As of now customer field does not seem to work in SynQ, additionally it requires the customer to be created beforehand and we don't seem to have support for creating it with API.
Therefore, I vote to have it removed from payload. Additionally, we should include address, even though it is only name and street address. SynQ seems to accept a shipping address with only those two fields (`contactPerson` and `addressLine1`):
![image](https://github.com/user-attachments/assets/80c62752-52fb-477d-a838-03a1e97d2c82)